app/models/service_template_provision_task.rb

```diff
@@ -169,6 +169,14 @@ def mark_pending_items_as_finished
     end
   end
 
+  def before_ae_starts(_options)
+    reload
+    unless state.to_s.downcase == "active"
```

Comment:
@bzwei @tinaafitz 
Can the state be something other than active, like a failed state, or retry?